Bookeun/Presentation/SelectExercise/View/SelectExerciseViewCell.swift

```diff
@@ -26,7 +38,17 @@ class SelectExerciseViewCell: UICollectionViewCell, Nameable {
         gradeViews.forEach({ $0.layer.cornerRadius = $0.bounds.height / 2 })
     }
     
-    @IBAction private func actionSelecte(_ sender: UIButton) {
+    @IBAction private func actionSelect(_ sender: UIButton) {
         sender.isSelected.toggle()
+        delegate?.didTapButton(exercise)
```

Comment:
```suggestion
        delegate?.didTapped(exercise)
```
Cell Tapped != Button Tapped는 서로 다른 의미이기에 delegate의 네이밍을 수정하면 좋을 것 같습니다.